test/distributed/org/apache/cassandra/distributed/test/cql3/AccordInteropSingleNodeTableWalkTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.test.cql3;
+
+import accord.utils.Property;
+import org.apache.cassandra.service.consensus.TransactionalMode;
+
+public class AccordInteropSingleNodeTableWalkTest extends SingleNodeTableWalkTest
```

Comment:
I think im going to drop all Interop tests for now as they are all broken... Ariel has a patch in-flight to fix known issues... so rather than blocking this patch its prob best to just skip testing accord for now...